boot-script/index.js

```diff
@@ -4,8 +4,12 @@
 // License text available at https://opensource.org/licenses/MIT
 
 'use strict';
-var helpers = require('../lib/helpers');
+
 var path = require('path');
+var SG = require('strong-globalize');
```

Comment:
Minor comment: no need to have `require('path')` before `require('strong-globalize`)`